atlp-rwanda/Team-furebo-E-commerce-fn
# #185172181 A user should be able to reset their password

### What does this PR do?
This PR adds the reset password functionality using a preexisting email address.
### Description of Task to be completed
In case a user forgets their password, they will be able to reset it using this functionality by way of their registration email.
This feature is composed of two separate pages a 'Request Password Reset' page where they will enter a valid email and receive a link that will redirect them to a second page 'Create New Password' page. From there, they will be able to enter a new password which will be validated and confirmed by double insertion in a confirm password field. Once the new password is verified and validated they will be redirected to the login where they will be able to access their account with the new password.
### How should this be manually tested?
* Navigate to the login page and click on the "Forgot Password" link.
* Input a preexisting valid email address associated with the account and Submit
* In your Email, click the link sent, and you will be redirected to a new page where you can enter your new password.
* Enter a new password and confirm it, make sure it complies with the validation terms, and Submit.
* If successful a message will be displayed confirming the change, and will be redirected to the login page.
* Finally access your account with the new password.
### What are the relevant pivotal tracker/Trello stories?
#185172181
